src/locale/locales/pt/messages.

```diff
@@ -0,0 +1,2468 @@
+msgid ""
+msgstr ""
+"POT-Creation-Date: 2023-12-22 01:56+0530\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"X-Generator: @lingui/cli\n"
+"Language: pt\n"
+"Project-Id-Version: lang-bluesky\n"
+"Report-Msgid-Bugs-To: \n"
+"PO-Revision-Date: 2023-12-23 16:51\n"
+"Last-Translator: \n"
+"Language-Team: Portuguese, Brazilian\n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+"X-Crowdin-Project: lang-bluesky\n"
```

Comment:
Are these crowdin flags necessary? (it's ok if they are!, just wondering)

Reply:
If you want to send the translation project there, people can collaborate. But I removed it anyway, take a look at crowdin.com remembering that the translation sent there is updated here on github using integration!

Fix